opm/common/OpmLog/EclipsePRTLog.cpp

```diff
@@ -56,4 +61,14 @@ namespace Opm {
         StreamLog::addTaggedMessage(Log::MessageType::Info, "", summary_msg);
     }
 
+    EclipsePRTLog::EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
Nitpick: hard to read, perhaps better to put one argument per line (also other constructor)?